### PR TITLE
Proposal tables and rules

### PR DESCRIPTION
**Pull request by chicoxyzzy**
[Rendered version](https://github.com/HolyJS-TCXX/proposals/blob/35ddfea3bdd5caf0e364b5137aef3e66febded73/README.md)

### CODE REVIEW

**Review by chicoxyzzy — COMMENTED** [review]
Note to self: добавить инфу о том, как проверить не существует ли уже такой пропозал

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# proposals
\ No newline at end of file
+# [HolyJS](https://holyjs-moscow.ru) TCXX proposals
+# [ECMAScript](https://github.com/tc39/ecma262) proposals
+
```
Comment (DmitryMakhnev) [inline README.md:4]: Надо вставить ссылку на описание https://github.com/HolyJS-TCXX/main

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+Proposal repo should be in English. Please read [how to write a good explainer](https://github.com/tc39/how-we-work/blob/master/explainer.md) recommendations from TC39.
+
+One can add maximum of 3 proposals per conference.
+
+After you've created proposal repo, please [file PR](https://github.com/HolyJS-TCXX/proposals/pulls) against this repo and add your proposal to one of the tables below.
+
+## Proposals
+
+- [Potential ECMAScript or Intl proposals](#potential-ecmascript-or-intl-proposals)
```
Comment (DmitryMakhnev) [inline README.md:26]: Добавить описание

> Potential proposals are helpful proposals which you really would like to see at next standards of ECMAScript or Intl. For example [`String.prototype.capitalize`](https://github.com/HolyJS-TCXX/proposal-to-capitalize-method-for-string-prototype)

> Just for fun proposals are funny proposals just for fun talking at party of the conference. For example [`🐈(cat) prefix to functions`](https://github.com/HolyJS-TCXX/HolyJS-Moscow-2018-TCXX/blob/master/proposal-example.md#ecmascript-proposal-add-cat-prefix-to-functions)

Comment (chicoxyzzy) [reply README.md:26]: helpful тут не подходит, надо какое-то другое слово. Кстати, potential возможно тоже

**Pull request merged**